nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSTransactionState.java

```diff
@@ -120,6 +121,10 @@
 
     public static final String READ_ACL_ASYNC_THRESHOLD_DEFAULT = "500";
 
+    public static final String NIN = "$nin";
+
+    public static final String IN = "$in";
```

Comment:
You can't have MongoDB-specific code in the generic DBS layer. Use integer constants or an enum instead. Define them in `DBSRepository`.

Reply:
As I didn't define a new api specifically to handle operators, I do the conversion in
`MongoDBConverter#valueToBson` which now handles operators.
I always call it in all queryKeyValue call. The value was not converted systematically before. This way, if we pass an operator in a single value, or in one of two values or even both, operators will be handled. 
What do you think ?